src/tests/ftest/util/test_utils_container.py

```diff
@@ -596,6 +596,8 @@ def write_objects(self, rank=None, obj_class=None):
                 self, self.record_qty.value, self.akey_size.value,
                 self.dkey_size.value, self.data_size.value, rank, obj_class,
                 self.data_array_size.value)
+        # Mark the pool as disconnected after API operations
+        self.pool.connected = False
```

Comment:
Is this needed? This could affect all tests using `write_objects`, not just this one test

Reply:
This is to take care of the failure of double pool disconnection.